_component_design/option-tiles.md

```diff
@@ -0,0 +1,338 @@
+---
+title: Option Tiles
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: >
+  An option tile is a selectable component that graphically represents a choice a user can make on a page.
+preview-image: preview-images/option-tile.svg
+status: stable
+resource: true
+last-modified: 2019-01-29
+helix-ui-css: false
+helix-ui-javascript: false
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Like radios and checkboxes, option tiles provide the user a means of selecting from a group of options. Because option tiles increase the selectable space of a choice, you can add an icon and description, which provides context and a richer information experience to the user. Option tiles help decision making by focusing the user's attention.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct option tiles:
+
+- Only use as many option tiles as you need. Too few option tiles (for example, two option tiles) can be visually confusing to the user while larger sets of option tiles increase the user's information processing demands and can lead to *option paralysis*. The number of option tiles is also a function of the option tile size and the user's screen size. We recommend using option tiles when there is more than one and a maximum of nine choices.
```

Comment:
1) this feels like a LOT of text. can we shorten it? 
Only use as many option tiles as you need. We recommend between 2 and 9 to avoid visual confusion or option paralysis.

Reply:
Done.